sdk/app_cpu1/common/sys/scheduler.h

```diff
@@ -9,10 +9,15 @@
 #include "usr/user_config.h"
 
 // Utility defines for time conversions
-#define USEC_IN_SEC       (1000000)
+#define USEC_IN_SEC       (1000000.0)
 #define SEC_TO_USEC(sec)  (sec * USEC_IN_SEC)
 #define USEC_TO_SEC(usec) (usec / USEC_IN_SEC)
 
+// Timing Threshold - 60ns
+//   Typically, the variance from the expected interval is no more than
+//   20ns, but we'll use a tolerance of three times this for safety :)
+#define SCHEDULER_INTERVAL_TOLERANCE (-0.06)
```

Comment:
Can you append `_USEC` to this so it is clearer? similar to the `task_control_block_t` members which record time, they have `_usec` appended

Reply:
Also, why is this negative?

...I just reviewed the logic above, and I agree this should be negative. Can you add a comment to explain here?